src/main/resources/db/migration/V202409041445__create_table_pets.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE pets (
+                      id SERIAL PRIMARY KEY,
+                      name VARCHAR(30) NOT NULL,
+                      type VARCHAR(15) NOT NULL,
+                      company_id BIGINT NOT NULL,
+                      date_of_arrival DATE DEFAULT CURRENT_DATE
+);
+
+CREATE INDEX idx_pets_with_company_id ON pets (company_id);
+
+
+INSERT INTO pets (name, type, company_id, date_of_arrival)
```

Comment:
if you want the queries wont be a comment than its better to separate it to a different migration file